docker-compose.yml

```diff
@@ -6,38 +6,56 @@ services:
         command: >
             /bin/bash -c "
                 until echo > /dev/tcp/postgres/5432; do sleep 1; done
-                npx pubsweet server
+                node app
             "
         ports:
             - 3000:3000
         depends_on:
             - postgres
+        env_file:
+          - .env
         environment:
-            NODE_ENV: development
+            NODE_ENV: production
             NODE_CONFIG: '{
-                "pubsweet-client": {
-                    "API_ENDPOINT": "/api"
-                },
-                "pubsweet-server": {
-                    "baseUrl": "${PUBSWEET_BASEURL}",
-                    "secret": "somesecret",
-                    "hostname": "${PUBSWEET_HOSTNAME}"
-                },
-                "password-reset": {
-                    "url": "${PUBSWEET_BASEURL}/password-reset"
-                },
-                "pubsweet-component-ink-backend": {
-                    "inkEndpoint": "https://demo--ink.elifesciences.org",
-                    "email": "fake",
-                    "password": "fake"
+                "scienceBeam":{
+                    "url":"http://sciencebeam:8075/api/convert"
                 },
                 "auth-orcid": {
-                    "clientID": "${ORCID_CLIENT_ID}",
-                    "clientSecret": "${ORCID_CLIENT_SECRET}"
+                    "sandbox": true
                 }
             }'
             PGHOST: postgres
             PGUSER: xpub
+
+    sciencebeam:
+        image: elifesciences/sciencebeam:${SCIENCEBEAM_TAG}
+        command: ./server.sh \
+            --host=0.0.0.0 --port=8075 \
+            --grobid-url http://grobid:8070/api
+        ports:
+        - "8075:8075"
+        networks:
+        - default
+        - grobid
+        restart: always
+        depends_on:
+        - grobid
+
+    grobid:
+        image: lfoppiano/grobid:${GROBID_TAG}
+        environment:
+        - JAVA_OPTS=-Xmx1g
```

Comment:
the default `t2.medium` server has 4GB in total; we may re-evaluate if more memory is needed later on

Reply:
I had to up the memory allocation for Docker itself to 4gb to get it to work, but it seems the container runs OK with this config.

Reply:
FTR: Daniel did mention that grobid can get memory hungry.